rpm/BUILD.bazel

```diff
@@ -721,6 +721,8 @@ rpmtree(
         "/var/run": "../run",
         "/usr/sbin/iptables": "/usr/sbin/iptables-legacy",
         "/usr/bin/nc": "/usr/bin/ncat",
+        # Create a symlink to OVMF binary with SEV support (edk2 rpm does not do that for unknown reason)
+        "/usr/share/OVMF/OVMF_CODE.cc.fd": "../edk2/ovmf/OVMF_CODE.cc.fd",
```

Comment:
It probably does it in a post-install task which we don't run. :)

Reply:
But it does symlink other OVMF binaries...